config/README.md

```diff
@@ -0,0 +1,5 @@
+# configuration 
+
+this folder contains the configurations files for maelstro
+
+we usually copy it into georchestra datadir
```

Comment:
Please mind the uppercased letters :-)

"we usually copy it into georchestra datadir" => "When deploying with geOrchestra, these files should be copied into a `maelstro` subfolder of the [geOrchestra datadir](https://github.com/georchestra/datadir)"